scss/civicrm/search/pages/_full-text-search.scss

```diff
@@ -2,9 +2,10 @@
 
 .CRM_Contact_Form_Search_Custom .crm-search-form-block table.form-layout-compressed {
   td {
-    input.crm-form-submit {
+    button.crm-form-submit {
       @extend .btn-sm;
       margin-left: 20px !important;
+      margin-top: -8px;
```

Comment:
are we trying to fix a new issue or an existent one? changing from inputs to button misaligned the element?

Reply:
Yes the position was changed after changing to button. Because we have a long tree of styles being applied from different themes/css file(reset.css, civicrm.css etc).

But I found a better way of fixing this, that is by adding
```
float: none;
vertical-align: initial;
```